resources/measures/upgrade_hvac_vrf_hr_doas/measure.rb

```diff
@@ -1077,7 +1155,7 @@ def run(model, runner, user_arguments)
       "RSD", # Restaurant - Sit-Down
       "Hsp", # Health/Medical - Hospital
       #'RetailStandalone',
-      #'RetailStripmall'
+      'RetailStripmall',
```

Comment:
change summary: excluding retail strip mall based on feedback from Eric B.

Reply:
After speaking with Eric, we should keep strip malls, but explain in the documentation that in practice, each tenant would likely implement their own VRF/ERV system independent of other tenants so they could be separately metered.